src/main/java/org/mqttbee/api/mqtt/mqtt3/Mqtt3ClientBuilder.java

```diff
@@ -33,17 +36,27 @@
     private final MqttClientIdentifierImpl identifier;
     private final String serverHost;
     private final int serverPort;
-    private final boolean usesSSL;
+    private final String serverPath;
     private final MqttClientExecutorConfigImpl executorConfig;
+    private final boolean usesWebSockets;
```

Comment:
Using MqttSslConfig and MqttWebsocketConfig makes it a lot cleaner

Reply:
yes, no longer needed